validation/go.mod

```diff
@@ -4,11 +4,11 @@ go 1.21
 
 replace github.com/ethereum-optimism/superchain-registry/superchain => ../superchain
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101304.1
+replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101308.2-rc.1.0.20240222161420-b2b2a2ce17d5
```

Comment:
Here's a strategy we could employ to merge all of the in-flight work sanely:

1. Temporarily disable CI checks for linting and testing in `validation`
2. (optional) undo the changes to `validation` module on this PR
3. Merge this PR 
4. Update downstream PRs with the merge commit hash*
5. Return to this repo and reenable CI checks, update this go.mod file accordingly

* this makes me wonder if it is appropriate to have a more formal "release" process in this repo.

Reply:
can't we just temporarily pin the op-geth commit of the PR that pulls in the superchain (also per temp commit)?

Reply:
so the op-geth pr would reference a commit that has a failed validation check, but then we update this pr to point to the op-geth pr that points to this old temp commit, so we can already test validation while all PRs are still open.

Reply:
Yes I think this is doable and essentially the state of things as of now. I believe the validation checks running on this PR give us confidence that the overrides are being applied correctly.

 I suppose I was worried that we will be merging this PR to `main` despite it referencing a temporary commit of `op-geth`. But I guess this isn't so bad and that only needs to be the case for a short while. Probably no worse than disabling validation for a short while.